app/views/vacancies/index.html.slim

```diff
@@ -4,7 +4,7 @@
 - elsif @landing_page.present?
   - content_for :page_title_prefix, t("landing_pages.heading", landing_page: @landing_page.humanize)
 - else
-  - content_for :page_title_prefix, t("home.index.title")
+  - content_for :page_title_prefix, t("vacancies.index.title", count: params[:page].to_i)
```

Comment:
this can be just:

`- content_for :page_title_prefix, t(".title", count: params[:page].to_i)`